cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,31 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import userPageObject from '../support/pages/user.pageObject';
+
+const userPage = new userPageObject();
+
 describe('User', () => {
-  before(() => {
+  let userForFollow;
+  let userFollower;
 
+  before(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then(generateUser => {
+      userForFollow = generateUser;
+    });
+    cy.task('generateUser').then(generateUser => {
+      userFollower = generateUser;
+    });
   });
 
   it.skip('should be able to follow the another user', () => {
+    cy.register(userForFollow.email, userForFollow.username, userForFollow.password);
```

Comment:
Avoid directly invoking commands. Refactor to follow PageObject pattern. Replace with a method from the `userPage` object, like `userPage.register(userForFollow)`. Do the same for `cy.login` command.